### PR TITLE
Add Date::Calc perl module

### PR DESCRIPTION
**Pull request by fritzkink**

### CODE REVIEW

File: components/perl/Date-Calc/Makefile [hunk 1]
```diff
@@ -0,0 +1,48 @@
...
+include $(WS_MAKE_RULES)/common.mk
+
+COMPONENT_TEST_TRANSFORMS += \
+        '-e "/^\#/d"' \
+        '-e "/^PERL/d"' \
+        '-e "/^make/d"'
+
+REQUIRED_PACKAGES += library/perl-5/carp-clan
+REQUIRED_PACKAGES += library/perl-5/bit-vector
+
```
Comment (AndWac) [inline components/perl/Date-Calc/Makefile:44]: PERL_REQUIRED_PACKAGES needed here?

Comment (fritzkink) [reply components/perl/Date-Calc/Makefile:44]: I tried initially but pkg5 did not get updated accordingly. So decided for REQUIRED_PACKAGES. But maybe  I did not understand correctly the PERL_ variant.

Comment (mtelka) [reply components/perl/Date-Calc/Makefile:44]: > I tried initially but pkg5 did not get updated accordingly. So decided for REQUIRED_PACKAGES. But maybe I did not understand correctly the PERL_ variant.

It should work.  Let me check...

Comment (mtelka) [reply components/perl/Date-Calc/Makefile:44]: I changed the `Makefile` and ran `gmake publish`.  Below is the result (as expected):

```
diff --git a/components/perl/Date-Calc/Makefile b/components/perl/Date-Calc/Makefile
index adf9edd2f..4303a4e91 100644
--- a/components/perl/Date-Calc/Makefile
+++ b/components/perl/Date-Calc/Makefile
@@ -39,8 +39,8 @@ COMPONENT_TEST_TRANSFORMS += \
         '-e "/^PERL/d"' \
         '-e "/^make/d"'
 
-REQUIRED_PACKAGES += library/perl-5/carp-clan
-REQUIRED_PACKAGES += library/perl-5/bit-vector
+PERL_REQUIRED_PACKAGES += library/perl-5/carp-clan
+PERL_REQUIRED_PACKAGES += library/perl-5/bit-vector
 
 # Auto-generated dependencies
 REQUIRED_PACKAGES += runtime/perl-522
diff --git a/components/perl/Date-Calc/pkg5 b/components/perl/Date-Calc/pkg5
index c3dddd244..077d823c0 100644
--- a/components/perl/Date-Calc/pkg5
+++ b/components/perl/Date-Calc/pkg5
@@ -2,7 +2,13 @@
     "dependencies": [
         "SUNWcs",
         "library/perl-5/bit-vector",
+        "library/perl-5/bit-vector-522",
+        "library/perl-5/bit-vector-524",
+        "library/perl-5/bit-vector-534",
         "library/perl-5/carp-clan",
+        "library/perl-5/carp-clan-522",
+        "library/perl-5/carp-clan-524",
+        "library/perl-5/carp-clan-534",
         "runtime/perl-522",
         "runtime/perl-524",
         "runtime/perl-534",
```

Comment (mtelka) [reply components/perl/Date-Calc/Makefile:44]: BTW, I started with exactly your dataset:

```
commit 742a78fe06f8a6bb3a5d24b60cd23e9d4a1c5316 (HEAD -> TEST)
Author: fritzkink <friedrich.kink@fkink.de>
Date:   Fri Jun 10 17:13:08 2022 +0200

    Add Date::Calc perl module
```

Comment (fritzkink) [reply components/perl/Date-Calc/Makefile:44]: ok tried once more and it seems that a reboot of my zone solved my issue. Works as you described. Thanks for the support.

**Pull request merged**